server/api.js

```diff
@@ -16,8 +16,25 @@ router.get("/", (req, res) => {
 	res.json({ message: "Hello, world!" });
 });
 
-router.get("/desks", (req, res) => {
-	res.json(desks);
+router.get("/desks", async (req, res) => {
+	try {
+		const deskResult = await db.query("SELECT * from desk;");
+		// convert array with objects having {username, booking_date} into {name,desk_id,desk,date}
+
+		const desks = deskResult.rows.map((row) => {
+			return {
+				id: row.id,
+				desk: row.desk_name,
+				xCoor:row.x,
+				yCoor:row.y,
```

Comment:
Please check the rule of naming here "xCoor" & "yCoor". It seems you used different rule in line 50 => "desk_id " !?

Reply:
this is already fixed in another pr. Many thanks for reviewing